go/cmd/mysqlctl/mysqlctl.go

```diff
@@ -21,76 +21,146 @@ import (
 	"context"
 	"flag"
 	"fmt"
-	"io"
 	"os"
 	"time"
 
-	"github.com/spf13/pflag"
+	"github.com/spf13/cobra"
 
-	"vitess.io/vitess/go/cmd"
+	"vitess.io/vitess/go/cmd/vtctldclient/cli"
```

Comment:
we should find a better home for this `cli.FinishedParsing` function, now that places besides `vtctldclient` will be using it.